pkg/lib-volume-populator/populator-machinery/controller.go

```diff
@@ -576,11 +578,23 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 
 		// If the pod doesn't exist yet, create it
 		if pod == nil {
+			isController := false
+
 			// Make the pod
 			pod = &corev1.Pod{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      podName,
 					Namespace: populatorNamespace,
+					Labels:    map[string]string{"pvcID": pvc.Name},
```

Comment:
it's ok to set the PVC as the owner below - in case the pod wasn't removed along with the migration, but I think it would simplify things it we correlate the pod with the migration as well / instead